plugin/src/main/scala-2.12/chisel3/internal/plugin/ChiselPlugin.scala

```diff
@@ -24,7 +25,8 @@ class ChiselPlugin(val global: Global) extends Plugin {
   override def init(options: List[String], error: String => Unit): Boolean = {
     for (option <- options) {
       if (option == arguments.useBundlePluginOpt) {
-        arguments.useBundlePlugin = true
+        val msg = s"'${arguments.useBundlePluginFullOpt}' is now default behavior, you can stop using the scalacOption."
```

Comment:
Can you write the actual scalacOption here in the message?

Reply:
That's what the string interpolator is doing!